### ISSUE TITLE
Why keras removed Graph model?

### ISSUE THREAD

**Issue by walleva** [fchollet/keras#2802]
I am implementing a model with triplet loss according to  [Recommendations in Keras using triplet loss](https://github.com/maciejkula/triplet_recommendations_keras).
I met some problems with Graph and my keras version is 1.0.1. So I went to the [keras.io](keras.io) document.
However, Graph has been removed at version 1.0.3 and related documents have also been removed.
It seems that the framework is changed to functional API.
So why removed Graph? If I still want to use Graph, could I find the past keras.io document? Thanks!

**Comment by cmyr** [comment]
Graph was deprecated in 1.0; it has been superceded by the new [functional API](http://keras.io/getting-started/functional-api-guide/).

**Comment by fchollet** [comment]
> could I find the past keras.io document? 

Yes, you can git checkout to the 0.3.3 tag, and you will find the documentation for that version under `docs/`.

**Comment by walleva** [comment]
Thank you!
